File: problems/algorithms/basic-numbers-series/js/response/numbers-reverse-odd-for.js
let result = '';

for (let number = 99; number >= 0; number -= 2) {
  const whiteSpace = number % 10 === 1 ? '\n' : ' ';
  const value = number < 10 ? `0${number}` : number;
  result += value + whiteSpace;
}

console.log(result);

File: src/pages/exercises/basic-numbers-series/codes/ruby/response/numbers-reverse-odd-for.rb
result = ""

(1..99).step(2).reverse_each {|number|
  whiteSpace = number % 10 == 1 ? "\n" : " "
  value =  number < 10 ? "0#{number}" : number
  result += "#{value}#{whiteSpace}"
}

puts result
